src/utils/components/Consoles/Consoles.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import Loading from '@kubevirt-utils/components/Loading/Loading';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Bullseye } from '@patternfly/react-core';
+
+import { AccessConsoles } from './components/AccessConsoles/AccessConsoles';
+import SerialConsoleConnector from './components/SerialConsole/SerialConsoleConnector';
+import { SERIAL_CONSOLE_TYPE, VNC_CONSOLE_TYPE } from './components/utils/ConsoleConsts';
+import VncConsole from './components/vnc-console/VncConsole';
+import { INSECURE, SECURE } from './utils/constants';
+import { isConnectionEncrypted } from './utils/utils';
+
+const Consoles: React.FC<ConsolesProps> = ({ vmi }) => {
```

Comment:
can `Consoles` receive an obj that can be VM or VMI?
if its a VM try to fetch its VMI?